config/routes.rb

```diff
@@ -1,3 +1,12 @@
 Rails.application.routes.draw do
+
+  get 'invitations/new'
+  post 'invitations/create'
```

Comment:
- [ ] Why don't you add a custom pathname to these routes? That makes it more convenient to use them in views and controllers.
[Read more here.](https://guides.rubyonrails.org/routing.html#generating-paths-and-urls-from-code)